core/src/main/scala/chisel3/RawModule.scala

```diff
@@ -70,7 +83,7 @@ abstract class RawModule(implicit moduleCompileOptions: CompileOptions) extends
               case MemoryPortBinding(_, _) =>
                 id.forceName(None, default = "MPORT", _namespace)
               case PortBinding(_) =>
-                id.forceName(None, default = "PORT", _namespace)
+                id.forceName(None, default = "PORT", _namespace, true, { x: String => ModuleIO(this, x) })
```

Comment:
```suggestion
                id.forceName(None, default = "PORT", _namespace, true, x => ModuleIO(this, x))
```
You don't need the `{}` and type. It's fine to include them if you think it makes it more clear, I don't but YMMV.